pkg/oauth/server/osinserver/defaults.go

```diff
@@ -24,6 +24,7 @@ func NewDefaultServerConfig() *osin.ServerConfig {
 	config.AllowClientSecretInParams = true
 	config.AllowGetAccessRequest = true
 	config.RedirectUriSeparator = ","
+	config.ErrorStatusCode = http.StatusInternalServerError
```

Comment:
The error we were actually generating (server_error) was 500.  Since all we have is a hammer....


> On Mar 30, 2015, at 2:36 PM, David Eads <notifications@github.com> wrote:
> 
> In pkg/oauth/server/osinserver/defaults.go:
> 
> > @@ -24,6 +24,7 @@ func NewDefaultServerConfig() *osin.ServerConfig {
> >  	config.AllowClientSecretInParams = true
> >  	config.AllowGetAccessRequest = true
> >  	config.RedirectUriSeparator = ","
> > +	config.ErrorStatusCode = http.StatusInternalServerError
> Shouldn't this be: StatusBadRequest as per: https://tools.ietf.org/html/rfc6749#section-5.2?
> 
> —
> Reply to this email directly or view it on GitHub.
>